exporter/clickhouselogsexporter/exporter.go

```diff
@@ -614,13 +668,21 @@ func (e *clickhouseLogsExporter) addAttrsToAttributeKeysStatement(
 
 func (e *clickhouseLogsExporter) addAttrsToTagStatement(
 	statement driver.Batch,
+	tagStatementV2 driver.Batch,
 	attributeKeysStmt driver.Batch,
 	resourceKeysStmt driver.Batch,
 	tagType utils.TagType,
 	attrs attributesToSliceResponse,
 	useNewSchema bool,
+	shouldSkipKeys map[string]shouldSkipKey,
 ) error {
+	unixMilli := (time.Now().UnixMilli() / 3600000) * 3600000
```

Comment:
any reason for unixmilli ? also what are you doing here ?

Reply:
I need an epoch timestamp, rounded to the nearest 1 hour, and I generally use epoch milli. today, we store one instance for the key=value. The timestamp column contains the latest timestamp. In this new table, we maintain one entry for 1 hour. This will help, that otherwise can't be easily done with the existing table.

- What is the hourly churn rate?
- Show only relevant values in the selected period. 

Since we eliminate keys with high cardinality values from this table, storage won't be a problem.

Reply:
can you add it as comment in code ? it will be helpful if someone looks at it.

Reply:
I intend to move this logic to the metadata exporter (https://github.com/SigNoz/signoz-otel-collector/issues/480, Do you have any concerns? I believe only main tables v2 and v2 resource logic should be part of the logs exporter), I will add it there.

Reply:
ok